dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1517,6 +1517,97 @@ sources:
                   config: *unique-conditions
                   meta:
                     description: yrblt should be between 1850 and now
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_205_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '205'
+                  meta:
+                    description: 205s should be > 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_206_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '206'
+                  meta:
+                    description: 206s should be > 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_207_less_than_62_years_old
+                  min_value: cast(year(current_date) as decimal) - 62
+                  max_value: cast(year(current_date) as decimal)
```

Comment:
**[Suggestion]** In a similar vein, our test for [overall age ranges of dwellings](https://github.com/ccao-data/data-architecture/blob/66442f5a33153e33de914421967181dc53839c54/dbt/models/iasworld/schema/iasworld.dweldat.yml#L1561-L1568) already checks to make sure that `yrblt` is less than the current year, so we can remove this range to avoid duplicate failures:

```suggestion
```